data/resource.go

```diff
@@ -2,15 +2,14 @@ package data
 
 import (
 	"fmt"
+	"github.com/laurent22/ical-go/ical"
```

Comment:
I guess this change is accidental, right?

Reply:
Yes, probably `go fmt` which messed things up a bit, I'll revert it.

Reply:
Fixed.